app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -570,7 +571,12 @@ class CommonTriageQuestionsController @Inject() (
                 representeeAnswers = None,
                 examplePropertyDetailsAnswers = None,
                 yearToDateLiabilityAnswers = None,
-                supportingEvidenceAnswers = None
+                supportingEvidenceAnswers = None,
+                exemptionAndLossesAnswers =
+                  if (furtherReturn)
+                    multiple.exemptionAndLossesAnswers.map(_.unset(_.inYearLosses).unset(_.previousYearsLosses))
```

Comment:
it would be better to just set the losses and exemptions section to `None` for a further return here and below so that the user sees the state `TO DO` on the task list instead of `IN PROGRESS`